src/models/Stop.js

```diff
@@ -0,0 +1,20 @@
+const { DataTypes, UUIDV4 } = require('sequelize')
+
+// Create & export TripAdditionalCharge model
+module.exports = (sequelize) => {
+  sequelize.define('Stop', {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: UUIDV4,
+    },
+    location: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    duration: {
```

Comment:
Better `time` instead of `duration`. Also and more important, why not `TIME` DataType here?, It's a tricky problem, in Postgres it should be an `INTERVAL` DataType, but is possible to configure it from sequelize?
In v7 there is a custom data type so we can configure to `INTERVAL`, but I'm not sure that works in v6.